app/src/main/res/layout/category_list.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:orientation="horizontal"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <TextView
+        android:id="@+id/text1"
```

Comment:
Either make a non-generic ID name or don't give it an ID if it isn't being referenced.

Reply:
This id was for testing purposes. Will modify.

Reply:
This id was for testing purposes. Will modify.